mindsdb_native/libs/helpers/general_helpers.py

```diff
@@ -58,19 +52,19 @@ def check_for_updates():
         ret = ret.json()
     except Exception as e:
         try:
-            log.warning(f'Got reponse: {ret} from update check server !')
+            log.warning(f'Got reponse: {ret} from update check server!')
         except Exception:
-            log.warning(f'Got no response from update check server !')
-        log.warning(f'Could not check for updates, got excetpion {e} !')
+            log.warning(f'Got no response from update check server!')
+        log.warning(f'Could not check for updates, got excetpion: {e}!')
         return
 
     try:
-        if 'version' in ret and ret['version']!= __version__:
-            log.warning("There is a new version of MindsDB {version}, please upgrade using:\npip3 uninstall mindsdb --upgrade".format(version=ret['version']))
+        if 'version' in ret and ret['version'] != __version__:
```

Comment:
Unless you modified the website at `http://check.mindsdb.com` this PR won't work, what you need to do is use the version of `mindsdb` not `mindsdb_native` here.

If there's no `mindsdb` installed just warn the user about something like `You are running on a machine with only mindsdb_native, a python library interface for Mindsdb, you might also wish to install the main mindsdb pip package`.

Reply:
This will be solved on the side of the telemetry server:
https://github.com/mindsdb/mindsdb_server_enterprise/pull/1